CHANGES.txt

```diff
@@ -1,4 +1,5 @@
 4.0-alpha4
+ * Fix cqlsh output test (CASSANDRA-15707)
```

Comment:
i'm not too sure we need non-user entries in `CHANGES.txt`

Reply:
not sure either...I can remove those as they cause merge conflicts all the time ;)

Reply:
there are a bunch of entries in CHANGES.txt that are for test fixes, so I did the same but personally either work for me

Reply:
Normally it is only user impacting changes. Not sure why people put them there.

Reply:
makes perfect sense to mention only user-relevant things in CHANGES.txt, so I removed my changes there